File: app/src/main/java/com/mnj/java_solutions/IsPrime.java
package com.mnj.java_solutions;

public class IsPrime {

    public void isPrimeNumber() {
        int n = 11;

        // Check from 2 to n/2
        for (int i = 2; i <= n / 2; i++)
            if (n % i == 0) {
                System.out.println("==>> " + n + " is NOT a prime number");
                return;
            } else {
                System.out.println("==>> " + n + " is a prime number");
                return;
            }
    }
}

File: app/src/main/java/com/mnj/java_solutions/IsPrime.kt
package com.mnj.java_solutions

class IsPrime {
    // Check from 2 to n/2
    val isPrimeNumber: Unit
        get() {
            val n = 11
            // Check from 2 to n/2
            for (i in 2..n / 2)
                if (n % i == 0) {
                println("==>> $n is NOT a prime number")
                return
            } else {
                println("==>> $n is a prime number")
                return
            }
        }
}